schema/resolversTests/validation.test.js

```diff
@@ -205,12 +206,61 @@ describe("resolvers", () => {
           inclusive: true
         }
       });
+
       expect(result).toMatchObject({
         id: currencyValidation.maxValue.id,
         custom: 10,
         inclusive: true
       });
     });
+
+    it("can update inclusive and previous answer max values", async () => {
+      const previousAnswer = await createNewAnswer(firstPage, "Number");
+      const currencyAnswer = await createNewAnswer(firstPage, "Currency");
+      const currencyValidation = await queryAnswerValidations(
+        currencyAnswer.id
+      );
+
+      const result = await mutateValidationParameters({
+        id: currencyValidation.maxValue.id,
+        maxValueInput: {
+          previousAnswer: previousAnswer.id,
+          inclusive: true
+        }
+      });
+
+      expect(result).toMatchObject({
+        id: currencyValidation.maxValue.id,
+        previousAnswer: {
+          id: previousAnswer.id
+        },
+        inclusive: true
+      });
+    });
+
+    it("can update inclusive and entity type", async () => {
+      const currencyAnswer = await createNewAnswer(firstPage, "Currency");
+      const currencyValidation = await queryAnswerValidations(
+        currencyAnswer.id
+      );
+
+      const entityTypes = ["Custom", "PreviousAnswer"];
+
+      entityTypes.map(entityType => {
```

Comment:
Should this be `forEach` and `await`ing each mutation?

Reply:
Also, it doesn't return the promise array - so would this fail?

Reply:
This works fine because `map` is synchronous.  Each request is resolved within the map so no need to return a promise array.